content/infrastructure/kinds-of-hubs.md

```diff
@@ -4,52 +4,60 @@ weight = 30
   columns = "1"
 +++
 
-# Kinds of hubs
+# Kinds of JupyterHubs that we support
 
-2i2c Hubs are extremely flexible and customizable, and can generally meet most
+JupyterHubs are flexible and customizable, and can generally meet most
 use-cases in research and education. There are 4 kinds of JupyterHub configurations
-we have found to be very common, and might help you think about what kinda JupyterHub
-configuration you want.
+we have found to be very common. We put extra emphasis on supporting these
+use-cases, which we'll describe below.
 
-## JupyterHub for your class
+## Teaching and learning with data science
```

Comment:
I was thinking of framing all these as 'JupyterHub for X', where X is a user persona of sorts. So folks should be able to tell if there's something that fits them from a scan. For example, both someone running a small class and someone with 20000 undergrads can fit 'teaching and learning with data science'